### ISSUE TITLE
[feature] Collapse / hide list

### ISSUE THREAD

**Issue by nmariano9** [mgmeyers/obsidian-kanban#629]
This would be great for being able to maintain a list of tasks to potentially be done, but keep them off your mental docket when you are looking at the kanban. I think this would be really helpful for managing stress and bandwidth (based on ideas from Cal Newport and Oliver Burkeman). Thanks!

**Comment by Hanawa02** [comment]
It would also be nice if your kanban is part of a note with more content and you don't want to separate them. For example, if I have a note for Project X, where I list/link some functionalities it has, but also want to have the tasks (Kanban) in the same file, I could hide the lists created by other content's sections.

**Comment by nmariano9** [comment]
No, I mean collapsing an entire list. I now see that there is an arrow that enables this!
